src/components/Router/StepsRouter.js

```diff
@@ -0,0 +1,95 @@
+import { h, Component } from 'preact'
+import createHistory from 'history/createBrowserHistory'
+
+import { events } from '../../core'
+import {sendScreen} from '../../Tracker'
+import {wrapArray} from '../utils/array'
+
+const history = createHistory()
+
+class StepsRouter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
this state is not needed, both `step` and `componentsList` always equals the props